{"name": "test_someting2", "status": "failed", "statusDetails": {"message": "AssertionError: The sataus code is wrong", "trace": "self = <tests.test.crud.test_crud.TestCrud object at 0x00000235DDC7C760>\nget_booking_id = 2679\n\n    def test_someting2(self,get_booking_id): ##Time pass ignore it\n        url=APIConstants.url_get_search(get_booking_id)\n        print(get_booking_id)\n        response=get_request(url=url,auth=None,in_json=False)\n        print(response.json())\n        actual_status_code=response.status_code\n>       verify_http_status_code(response_data=actual_status_code,expected_data=404)\n\ntest_crud.py:83: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresponse_data = 200, expected_data = 404\n\n    def verify_http_status_code(response_data, expected_data):\n>       assert response_data == expected_data ,\"The sataus code is wrong\"\nE       AssertionError: The sataus code is wrong\n\n..\\..\\..\\src\\helpers\\common_verification.py:2: AssertionError"}, "attachments": [{"name": "stdout", "source": "1f9455b4-7b8a-4fc7-aa36-cc575ec6c759-attachment.txt", "type": "text/plain"}], "start": 1730195767655, "stop": 1730195768668, "uuid": "218e8041-b95c-4bdd-9e10-0000e320da24", "historyId": "f96621bee909390f9e3b4e6550927874", "testCaseId": "78c38a059a447b210768886c0f3862a4", "fullName": "tests.test.crud.test_crud.TestCrud#test_someting2", "labels": [{"name": "parentSuite", "value": "tests.test.crud"}, {"name": "suite", "value": "test_crud"}, {"name": "subSuite", "value": "TestCrud"}, {"name": "host", "value": "LAPTOP-J82A4UMN"}, {"name": "thread", "value": "9220-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test.crud.test_crud"}]}